src/app/portlets/dot-templates/dot-template/dot-template.component.ts

```diff
@@ -48,4 +49,19 @@ export class DotTemplateComponent implements OnInit {
                 }
             });
     }
+
+    containerChange(container: DotContainer): void {
+        const selection = this.editor.getSelection();
+
+        const regex = new RegExp('//' + container.parentPermissionable.hostname);
+        const id = container.identifier.includes(container.parentPermissionable.hostname)
```

Comment:
We should get the `property` here as part of the object, if not, we have to double check.